src/UI/Components/Diagram/halo.ts

```diff
@@ -123,7 +130,7 @@ const createHalo = (
     const shapes = paper.findViewsInArea(area);
 
     shapes.map((shape) => {
-      highlighters.mask.remove(shape), "available-to-connect";
+      dia.HighlighterView.get(shape, "available-to-connect")?.remove();
```

Comment:
I'm not a fan of optional chaining to call methods on the potentially undefined/null variable. In this case, it might return an undefined (perhaps that's what you need), and also fail silently. Which can make it hard to debug at some point.

Reply:
I'll change it, although, here it is possible undefined/null, but that mean there is no highlighter given highlighter which is fine, as we are iterating through all of instances that are on the canvas.